src/warehouse/util.js

```diff
@@ -46,12 +47,16 @@ const timestampRegex = new RegExp(
 );
 
 function validTimestamp(input) {
-  if (timestampRegex.test(input)) {
-    // check if date value lies in between min time and max time. if not then it's not a valid timestamp
-    const dateInMs = Date.parse(new Date(input).toISOString());
-    if (minTimeInMs <= dateInMs && dateInMs <= maxTimeInMs) {
-      return true;
+  try {
+    if (timestampRegex.test(input)) {
+      // check if date value lies in between min time and max time. if not then it's not a valid timestamp
+      const dateInMs = Date.parse(new Date(input).toISOString());
```

Comment:
not js expert, but this could also work, without the need of try/catch
```suggestion
      const d = new Date(input)
      if (d.isNaN(d)) {
         return false
      }
      const dateInMs = Date.parse(d.toISOString());
```